taxi/models.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self: 'Manufacturer') -> str:
+        return f"{self.name}, {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    def __str__(self: 'Driver') -> str:
+        return f"{self.first_name} {self.last_name}: {self.license_number}"
```

Comment:
Avoid overusing f-strings when they are not necessary. You can simply return `self.first_name + ' ' + self.last_name + ': ' + self.license_number` instead.